### PR TITLE
fix: stop using class extensd as it breaks rollup

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: lib/jasmine/jasmine.ts [hunk 1]
```diff
@@ -35,43 +40,43 @@
     let originalJasmineFn: Function = jasmineEnv[methodName];
     jasmineEnv[methodName] = function(description: string, specDefinitions: Function) {
       return originalJasmineFn.call(this, description,  wrapDescribeInZone(specDefinitions));
-    } 
+    }
```
Comment (vicb) [inline lib/jasmine/jasmine.ts]: missing ; also below

File: lib/jasmine/jasmine.ts [hunk 2]
```diff
@@ -35,43 +40,43 @@
...
   });
   ['it', 'xit', 'fit'].forEach((methodName) => {
     let originalJasmineFn: Function = jasmineEnv[methodName];
     jasmineEnv[methodName] = function(description: string, specDefinitions: Function, timeout: number) {
       arguments[1] = wrapTestInZone(specDefinitions);
       return originalJasmineFn.apply(this, arguments);
-    } 
+    }
```
Comment (vicb) [inline lib/jasmine/jasmine.ts]: here

File: lib/jasmine/jasmine.ts (hunk 1 above)
Comment (mhevery) [inline lib/jasmine/jasmine.ts]: fixed

File: lib/jasmine/jasmine.ts (hunk 2 above)
Comment (mhevery) [inline lib/jasmine/jasmine.ts]: fixed

**Pull request merged**